include/GL/glui.h

```diff
@@ -442,7 +444,7 @@ class GLUIAPI GLUI_StdBitmaps
 {
 public:
     GLUI_StdBitmaps(); 
-    ~GLUI_StdBitmaps();
+    ~GLUI_StdBitmaps() = default;
 
```

Comment:
I have a work-in-progress branch to replace GLUI_CB with `std::function`.  It touches a lot of code, isn't production ready, but seems like a promising approach.  So my aim is to persuade @josch  to burn the bridge - C++11 or bust for GLEW 3.0.

https://github.com/nigels-com/glui/commit/bfc0552cd9858b64d75228834fb90e14bc20f566

It has the effect of bringing the widget creation and runtime behaviour together into the same block of code, an eliminates the integer IDs being passed around, stored, and fed back to the callbacks.  It has quite a bit of merit, in my opinion.

```
  new GLUI_Checkbox( light1, "Enabled", &light1_enabled, [&]() 
    { 
      if (light1_enabled)
      {
        glEnable( GL_LIGHT1 );
        light1_spinner->enable();
      }
      else
      {
        glDisable( GL_LIGHT1 ); 
        light1_spinner->disable();
      }
    });

```